indico/modules/events/abstracts/forms.py

```diff
@@ -192,7 +192,8 @@ class EditEmailTemplateRuleForm(IndicoForm):
     title = StringField(_("Title"), [DataRequired()])
     rules = EmailRuleListField(_("Rules"), [DataRequired()])
     stop_on_match = BooleanField(_("Stop on match"), [DataRequired()], widget=SwitchWidget(), default=True,
-                                 description=_("Do not evaluate any other rules once this one matches."))
+                                 description=_("If any of the rules from this email template matches, do not "
```

Comment:
It's [not incorrect](https://www.quora.com/If-I-begin-a-sentence-with-If-any-one-of-you-should-the-following-linking-verb-be-is-or-are) to say `matches`, but in this particular case I would agree that `match` fits better